### ISSUE TITLE
A11y_Azure Tools Storage Explorer_Blob Container Table_Table_ScreenReader: When the voiceover arrow focus moves to table data, the table data that is being announced are not associated with their column headers, row numbers and column numbers.

### ISSUE THREAD

**Issue by bbarik06** [microsoft/AzureStorageExplorer#5766]
**GitHubTags:** #A11yMAS;#A11yTCS;#A11ySev2;#BM_AzureToolsStorageExplorer_May2022;#Azure Tools Storage Explorer;#MAC;#FTP;#DesktopApp;#WCAG4.2.3;#Voiceover;#AINotInScope;

## Environment Details:
Application Name: Azure Tools Storage Explorer 
Application Version: 1.24.0
Mac Version: macOS Monterey 12.3.1
Screen Reader: Voice over 

## Repro Steps:
1. Please do not close this bug. This bug should only be closed by TCS, C+AI Accessibility tester after verification of the fix.
2. Launch Azure Tools Storage Explorer.
3. Sign in to Azure.
4. Tab to 'Explorer' pane tree view.
5. Navigate to any of the file inside "Blob Containers"
6. Tab till 'Name' column header of the table.
7. Tab till table data.
8. Turn on voiceover and observe.

## Actual:
When the voiceover arrow focus moves to table data, the table data that is being announced are not associated with their column headers, row numbers and column numbers.

## Expected:
When the voiceover arrow focus moves to table data, the table data that is being announced should be associated with their column headers, row numbers and column numbers.

## User Impact:
Screen reader users will face difficulty in understanding the table data structure if not associated with proper column header, row number, column number.

## Attachment:
![11_Table data](https://user-images.githubusercontent.com/98393966/169970031-a82fb12e-e593-480a-bffa-6bc58f06132c.gif)

**Comment by JasonYeMSFT** [comment]
Could you try using the normal keyboard to navigate in the table? Voice Over doesn't recognize the table element in html nor the aria-label we give to the element so there is nothing we can do to make it announce the correct content.

**Comment by bbarik06** [comment]
@JasonYeMSFT   This bug is specifically for screen reader users. Hence, we need to use keyboard + Screen readers to check the accessibility issue.

**Comment by JasonYeMSFT** [comment]
I have double checked the expected behavior of Voice Over. It will announce the row index when it navigates to the beginning of a row and will announce the cell content, column name and column index while navigating to each cell.

**Comment by bbarik06** [comment]
@JasonYeMSFT Could you please post the video recording the same here for to see!

**Comment by JasonYeMSFT** [comment]
https://user-images.githubusercontent.com/39359541/174853552-2a8560e9-48b7-4635-88c1-fdd2377a0ab6.mov

**Comment by bbarik06** [comment]
@JasonYeMSFT Could you please provide video sample of test being conducted in the live application and also please provide us the sample app you used to test for us to verify!

**Comment by JasonYeMSFT** [comment]
@bbarik06 We are still in the progress of fixing other the bugs and we will provide you the build when that's done.

**Comment by JasonYeMSFT** [comment]
I have switched to make the table use "grid" role, which is meant for interactive table elements. Now Voice Over should announce the "column x of y" together with the cell contents when navigating through the cells.

**Comment by bbarik06** [comment]
@JasonYeMSFT Is the bug fixed? If fixed, please provide the latest build.

**Comment by MRayermannMSFT** [comment]
Hi @bbarik06

Please verify that this bug is fixed using the Storage Explorer build at: https://storageexplorerpublish.blob.core.windows.net/privatebuilds/20220624.3-accessibility-testing/StorageExplorer-darwin-x64.zip

**Comment by bbarik06** [comment]
Issue is not repro now. Hence, closing the bug. Please refer the attachment.
Version: 1.25.0-dev
<img width="1440" alt="5766_Fixed" src="https://user-images.githubusercontent.com/98393966/176663526-e78f3460-b87d-4448-bed3-4e9a080f042e.png">
